dialogue-pizzaconundrum.js

```diff
@@ -34,7 +34,7 @@ const PIZZAMOOD_SAD = [
   {
     suspicion: ANYSUS,
     segments: [
-      "hes: #GiveMAriaAHug 🫂❤️",
+      "hes: #GiveMAriaAHug ❤️",
       "bibop: sad story... 😿",
       "thelibrarian: No pizza for the brave ☹️",
       "hes: Poor Maria 🌧️☹️☁️",
```

Comment:
```suggestion
      "hes: Poor Maria 🌧️🙁☁️",
```
How does this look? Or keep it the same

Reply:
It looks exactly the same to me, so if it looks better for you it's a win!

Reply:
Oh wait, no, it's less frowny. Doesn't matter, let's keep the one that works for you